data/scenario-03.json

```diff
@@ -139,7 +139,7 @@
                       "type": "rawImage",
                       "model": {
                         "width": 640,
-                        "height": 420,
+                        "height": 360,
```

Comment:
```
"width": 1024,
"height": 576,
```

Reply:
I tried to add images in various sizes so as to cover different users' devices. Let me know if you'd prefer smaller images :)